{
    "error": {
        "http_code": 404,
        "message": "Пользователь \"newISCSIUser\" не найден в кластере \"3\"",
        "traceback": [
            "Traceback (most recent call last):\n",
            "  File \"/usr/libexec/vstorage-ui-backend/venv/lib64/python3.4/site-packages/vstorage_ui_backend-1.3.185-py3.4.egg/backend/presentation/decorators.py\", line 28, in wrapper\n    return func(*args, **kwargs)\n",
            "  File \"/usr/libexec/vstorage-ui-backend/venv/lib64/python3.4/site-packages/vstorage_ui_backend-1.3.185-py3.4.egg/backend/logg.py\", line 127, in wrapper\n    return func(*args, **kwargs)\n",
            "  File \"/usr/libexec/vstorage-ui-backend/venv/lib64/python3.4/site-packages/vstorage_ui_backend-1.3.185-py3.4.egg/backend/presentation/iscsi/endpoints.py\", line 80, in put\n    user = IscsiPresentation.change_user(cluster_id, username, **data)\n",
            "  File \"/usr/libexec/vstorage-ui-backend/venv/lib64/python3.4/site-packages/vstorage_ui_backend-1.3.185-py3.4.egg/backend/business/models/iscsi/presentation.py\", line 552, in change_user\n    cluster_id, username, password, is_enabled, description\n",
            "  File \"/usr/libexec/vstorage-ui-backend/venv/lib64/python3.4/site-packages/vstorage_ui_backend-1.3.185-py3.4.egg/backend/business/models/iscsi/presentation.py\", line 515, in _validate_change_user_params\n    cls._get_user_ob(cluster_id, username)\n",
            "  File \"/usr/libexec/vstorage-ui-backend/venv/lib64/python3.4/site-packages/vstorage_ui_backend-1.3.185-py3.4.egg/backend/business/models/iscsi/presentation.py\", line 399, in _get_user_ob\n    username=username, cluster_id=cluster_id,\n",
            "backend.business.models.exceptions.ObjectDoesNotExist: Пользователь \"newISCSIUser\" не найден в кластере \"3\"\n"
        ],
        "subsytem": "presentation",
        "errors": null
    },
    "message": "Пользователь \"newISCSIUser\" не найден в кластере \"3\" (Deprecated field)"
}